### ISSUE TITLE
EISDIR error when trying to save file on network drive

### ISSUE THREAD

**Issue by comath11** [Microsoft/vscode#37048]
- VSCode Version: Code 1.17.2 (b813d12980308015bcd2b3a2f6efa5c810c33ba5, 2017-10-16T13:59:46.104Z)
- OS Version: Windows_NT x64 10.0.10586
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
vsc-material-theme|Equ|1.0.5
cpptools|ms-|0.14.0
reloaded-cpp|rel|0.1.6
partial-diff|ryu|0.4.0
ayu|tea|0.8.0

(14 theme extensions excluded)

---

When trying to save a file on a network drive, I get the error message "Failed to save 'tcpip.c': A system error occurred (EISDIR: illegal operation on a directory, open 'z:\cody\mledit\tcpip.c')".  Creating a new file and trying to save it to the drive also results in the same error.  This happens 100% of the time, not just occasionally.  I don't have this issue with any other editors, so I don't believe it is an issue with the network drive configuration.  However, the network drive is pretty unique, so there could be some weird compatibility issue or something.  I can open and view the files in the explorer; I just can't save them.  Let me know if I need to provide additional information.

Steps to Reproduce:

1. Open file on network drive or create a new file.
2. Modify file.
3. Attempt to save the file to the network drive.

This is what is logged for the error:
Failed to save 'tcpip.c': A system error occurred (EISDIR: illegal operation on a directory, open 'z:\cody\mledit\tcpip.c')
e.doShow @ messageService.ts:125
e.show @ messageService.ts:104
e.onSaveError @ saveErrorHandler.ts:181
t.onSaveError @ textFileEditorModel.ts:837
(anonymous) @ textFileEditorModel.ts:749
v @ winjs.base.raw.js:1209
enter @ winjs.base.raw.js:901
_run @ winjs.base.raw.js:1068
_chainedError @ winjs.base.raw.js:1031
n @ winjs.base.raw.js:736
then @ winjs.base.raw.js:1436
enter @ winjs.base.raw.js:747
_run @ winjs.base.raw.js:1068
_setCompleteValue @ winjs.base.raw.js:1052
v @ winjs.base.raw.js:1209
enter @ winjs.base.raw.js:901
_run @ winjs.base.raw.js:1068
_error @ winjs.base.raw.js:1041
done @ winjs.base.raw.js:1378
v @ winjs.base.raw.js:1224
enter @ winjs.base.raw.js:901
_run @ winjs.base.raw.js:1068
_error @ winjs.base.raw.js:1041
(anonymous) @ async.ts:675
(anonymous) @ extfs.ts:347
(anonymous) @ C:\Users\omathcor\Downloads\VSCode-win32-x64-1.17.2\resources\app\node_modules\graceful-fs\graceful…:241
(anonymous) @ original-fs.js:114

Reproduces without extensions: Yes

**Comment by comath11** [comment]
Yes, I just installed the latest stable build and I still get the same error.

**Comment by bpasero** [comment]
Can you try to reproduce with our nightly insider builds? You can give our preview releases a try from: https://code.visualstudio.com/insiders/

**Comment by comath11** [comment]
I know this is late and the issue is already closed, but I have installed the latest insiders build, and the issue still occurs.
